help/help.adoc

```diff
@@ -1,448 +1,301 @@
+|Key |Key of the object to download.
+|Download Location |Required. Path of the download location (for example, `/path/to/downloadLocation` or `C:\path\to\downloadLocation`).
 |===
 
-==== Output
-
-After the job runs, you can view the results on the Job Details page in {PRODUCT}. Every job step was completed successfully.
-
-In the *DownloadObject* step, select the *Log* button to see the diagnostic information.
-
-{PRODUCT} stores the key names and the download locations of the objects in property sheet.(Default location is */myJob/S3Output*.)
-
 
 [[ListBucket]]
-
-
 === ListBucket
 
-This procedure lists all the buckets.
+Lists all buckets. {PRODUCT} stores the list of buckets in the property sheet. The default location is `/myJob/S3Output`.
 
-==== Input
-
-. Go to the *ListBucket* procedure.
-. Enter the following parameters:
+==== ListBucket input parameters
 
+.ListBucket input parameters
 [cols="1a,1a",options="header"]
 |===
 |Parameter |Description
-|Configuration |The name of the configuration which holds all the connection information. This must reference a valid existing configuration. (Required)
+|Configuration |Required. The name of the configuration that holds all the connection information. This must reference a valid existing configuration.
 |===
 
-
-==== Output
-
-After the job runs, you can view the results on the Job Details page in {PRODUCT}. Every job step was completed successfully.
-
-In the *ListBucket* step, select the *Log* button to see the diagnostic information.
-
-{PRODUCT} stores the list of buckets in the property sheet (Default location is */myJob/S3Output*) as follows:
-
 [[ListFolder]]
-
-
 === ListFolder
 
-This procedure lists the contents of the folders, either recursively or nonrecursively.
-
-==== Input
+Lists the contents of the folders, either recursively or non-recursively. {PRODUCT} stores the list of all the objects in the folder in the property sheet. The default location is `/myJob/S3Output`.
 
-. Go to the *ListFolder* procedure.
-. Enter the following parameters:
+==== ListFolder input parameters
 
+.ListFolder input parameters
 [cols="1a,1a",options="header"]
 |===
 |Parameter |Description
-|Configuration |The name of the configuration which holds all the connection information. This must reference a valid existing configuration. (Required)
-|Bucket Name |Name of the bucketof which to list the folders. (Required)
+|Configuration |Required. The name of the configuration that holds all the connection information. This must reference a valid existing configuration.
+|Bucket Name |Required. Name of the bucket in which to list the folders.
 |Folder Name |Name of the folder or prefix to include in the list.
-|List Objects in this folder or Include all sub folders? |If selected, all objects in this folder and all subfolders will be in the list.
+|List Objects in this folder or Include all sub folders? |If selected, all objects in this folder and all sub-folders are included in the list.
```

Comment:
```suggestion
|List Objects in this folder or Include all subfolders? |If selected, all objects in this folder and all subfolders are included in the list.
```
I don't think this should be hyphenated.  Refer to https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/sub. Also, there is a space in `sub folders` in the label.

Reply:
Yeah, I was debating this change as well. Thanks!